### ISSUE TITLE
Some Auctionator Settings not Working after Latest Blizz Reset

### ISSUE THREAD

**Issue by Galthariel** [Auctionator/Auctionator#800]
After latest Reset neither the 48 Hours AH Duration or the Undercut are working. Tried to restart, relog the game but nothing worked. Full Scan worked just fine so I'm guessing this is something related to an update Blizz made.

**Comment by plusmouse** [comment]
The latest update to Auctionator moved the duration and undercutting stuff to the "Selling" tab - hijacking the blizzard tab caused some throttling issues, so we had to move the features.

**Comment by plusmouse** [comment]
See https://github.com/Auctionator/Auctionator/wiki/8.3.2-Release

**Comment by Galthariel** [comment]
Both Undercut and Duration are not working in the Selling Tab. Sorry if I forgot to mention that before.

**Comment by plusmouse** [comment]
Do the settings show correctly on the settings page?

**Comment by Galthariel** [comment]
Never mind and you're right. I was doing everything wrong. Since is easy for me to look at the bags instead of the bag view, I was right clicking the item in my bag and I did not noticed the tab changed from the TAB Selling to the Sell TAB and that's why I made a suggestion to include an option in the bag view to ignore items that are in our bags and we don't want to sell. It will reduce the amount of items you will see in the bag view making easy to work with the Bag view. Not a big deal though since I learned that I just alt click the item in my my bag (while in the Selling TAB) and everything will work as it should, both undercut and duration.

thanks much and I'm sorry again.
